setup.py

```diff
@@ -13,7 +13,7 @@
 install_requirements = [
     "boto3>=1.3,<2.0",
     "click==6.7",
-    "PyYaml==3.13",
+    "PyYaml==5.3.1",
```

Comment:
might be better to just match awscli, "PyYAML>=3.10,<5.3"?   https://github.com/aws/aws-cli/blob/ec0660a86df9f13e2f1465d30ea8fd62ceca3e95/setup.py#L35

Reply:
Main reason of my change is to prevent using vulnerable PyYAML. Requiring PyYAML older that 5.3.1 does not fulfill this.

Reply:
Anyway, your suggestion applied.